### PR TITLE
solved #911

### PR DESCRIPTION
**Pull request by utkarshnath**
solved #911 and removed the splash activity as its not being used now.

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/activities/SplashActivity.java [hunk 1]
```diff
@@ -1,32 +0,0 @@
-package org.fossasia.openevent.activities;
```
Comment (the-dagger) [inline android/app/src/main/java/org/fossasia/openevent/activities/SplashActivity.java]: Why did you remove the splash activity?
Even though it doesn't serves any meaningful purpose as of now, but we can use this for prefetching the session data and other time consuming tasks on app launch.

**Comment by utkarshnath** [comment]
@the-dagger added back the splash Activity
